Sources/Plasma/NucleusLib/pnMessage/plClientMsg.h

```diff
@@ -113,9 +117,9 @@ class plClientMsg : public plMessage
 
     void AddRoomLoc(plLocation loc);
 
-    // Used for kLoadAgeKeys, kLetGoOfAgeKeys only
-    ST::string  GetAgeName() const { return fAgeName; }
-    void        SetAgeName(const ST::string& age) { fAgeName = age; }
+    // Used for kLoadAgeKeys, kLetGoOfAgeKeys, kAddPage, and kDropPage
+    const ST::string& GetPath() const { return fPath; }
```

Comment:
This should return a `ST::string` copy, not a reference.  In practice, it shouldn't matter much, but there are some cases where it does, and it's better to be consistent to avoid those cases (IMO).